core/src/test/java/io/jenkins/pluginhealth/scoring/probes/CodeCoverageProbeTest.java

```diff
@@ -112,7 +112,7 @@ public void shouldBeInErrorWhenRepositoryIsNotInOrganization() {
 
         when(ctx.getUpdateCenter()).thenReturn(new UpdateCenter(
             Map.of(
-                pluginName, new io.jenkins.pluginhealth.scoring.model.updatecenter.Plugin(
+                pluginName, io.jenkins.pluginhealth.scoring.model.updatecenter.Plugin.of(
```

Comment:
Looks like a lot of those `Plugin` objects are the same, with similar values. Maybe extract that in a variable that can be reused everywhere?

Reply:
The [JavaDoc ](https://docs.oracle.com/en/java/javase/11/docs/api/java.net.http/java/net/http/HttpRequest.Builder.html#timeout(java.time.Duration))says 

>  The effect of not setting a timeout is the same as setting an infinite Duration, i.e. block forever.

I was not really sure which value to pick. Honestly, the APIs I had tested took less than a second to respond. So I figured 5 seconds is apt.

What should I actually write as the comment?

Reply:
Here  parameters like `pluginName`, `scmLink`, `defaultBranch` are mocked before being passed. 

You do not mean creating a constructor in Record like I already did it once?

Reply:
That's OK like this